server/routes/statistics.js

```diff
@@ -242,36 +237,16 @@ module.exports = function (config) {
 		}),
 		'get-flow-rejections-data': function (unresolvedQuery) {
 			return rejectionsQueryHandler.resolve(unresolvedQuery)(function (query) {
-				return getData(driver)(function (data) {
-					var fullSize;
-
-					data = sortData(
-						filterBusinessProcesses(data.businessProcesses, assign({
-							flowStatus: 'rejected'
-						}, query)),
-						function (bpA, bpB) {
-							return bpA.rejectedDateTime - bpB.rejectedDateTime;
-						}
-					);
-
-					if (!data.length) {
-						return { size: 0, view: [] };
-					}
-
-					fullSize = data.length;
-
-					data = getPage(data, query.page);
-
-					return getViewRecords(
-						data,
-						rejectionsListProperties,
-						rejectionsListComputedProperties
-					)(function (result) {
-						result.size = fullSize;
-
-						return result;
-					});
-				});
+				var data = { rows: [], pageCount: 1 };
+				data.pageCount = getPage(data.rows, query.page);
+				/* TODO inflate data.rows with real results
+					expected structure:
+					[
+					['Reason 1, Reason 2', '*', 3, 'John Smith', 'Revision', '02/03/2017', 'Super inc.'],
```

Comment:
Let's maybe use hashes instead of arrays of values that don't transfer any information about the meaning of each value.